test/evergreen/gcp_auth.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "axial-radius-373703",
+  "private_key_id": "gcp_private_id",
+  "private_key": "gcp_private_key",
+  "client_email": "sona-pm2681@axial-radius-373703.iam.gserviceaccount.com",
+  "client_id": "110952788608209299273",
```

Comment:
The following probably ought to be Evergreen variables:
* `project_id`
* `client_email`
* `client_id`
* `client_x509_cert_url`

Unless there's documentation stating these are safe to make public. If it looks internal, I think the onus of proof should be to prove that it's safe, not the other way around.

Reply:
Done! [21c567a](https://github.com/wiredtiger/wiredtiger/pull/8842/commits/21c567a58781deac536487c50a4b64fedf9565e4)